flink-runtime/src/main/java/org/apache/flink/runtime/clusterframework/FlinkResourceManager.java

```diff
@@ -113,7 +111,8 @@
 	/** The service to find the right leader JobManager (to support high availability) */
 	private final LeaderRetrievalService leaderRetriever;
 
-	/** The currently registered resources */
+	/** The resources which have been registered at the ResourceManager. This indicates the set
+	 * of containers where a TaskManager has been started and registered at a JobManager.  */
```

Comment:
We should think about whether it denotes the startup and registration or simply the startup of a TaskManager process. This depends on whether we send the RegisterResource message before or after TaskManager registration at the JobManager.